reference/docs-conceptual/getting-started/fundamental/Getting-Detailed-Help-Information.md

```diff
@@ -1,170 +1,203 @@
+## Getting conceptual help
 
-To display a list of conceptual topics, type:
+The Get-Help cmdlet also displays information about conceptual articles in PowerShell,
+including articles about the PowerShell language. Conceptual Help articles begin with the
+"about_" prefix, such as about_line_editing. (The name of the conceptual article must be entered in
+English even on non-English versions of PowerShell.)
 
-```
-get-help about_*
+To display a list of conceptual articles, type:
+
+```powershell
+Get-Help about_*
 ```
 
-To display a particular Help topic, type the topic name, for example:
+To display a particular Help article, type the article name, for example:
 
-```
-get-help about_command_syntax
+```powershell
+Get-Help about_command_syntax
 ```
 
-The parameters of Get-Help, such as *Detailed*, *Parameter*, and *Examples*, have no effect on the display of conceptual Help topics.
+The parameters of Get-Help, such as **Detailed**, **Parameter**, and **Examples**, have no effect
+on the display of conceptual Help articles.
 
-## Getting Help About Providers
-The Get-Help cmdlet displays information about Windows PowerShell providers. To get Help for a provider, type "Get-Help" followed by the provider name. For example, to get Help for the Registry provider, type:
+## Getting help about providers
 
-```
-get-help registry
+The Get-Help cmdlet displays information about PowerShell providers. To get Help for a
+provider, type `Get-Help` followed by the provider name. For example, to get Help for the Registry
+provider, type:
+
+```powershell
+Get-Help registry
 ```
 
-To get a list of all the provider Help topics in your session, type
+To get a list of all the provider Help articles in your session, type
 
-```
-get-help -category provider
+```powershell
+Get-Help -Category provider
 ```
 
-The parameters of Get-Help, such as *Detailed*, *Parameter*, and *Examples*, have no effect on the display of provider Help topics.
+The parameters of `Get-Help`, such as **Detailed**, **Parameter**, and **Examples**, have no effect
+on the display of provider Help articles.
 
-## Getting Help About Scripts and Functions
-Many scripts and functions in Windows PowerShell have Help topics. Use the Get-Help cmdlet to display the Help topics for scripts and functions.
+## Getting help about scripts and functions
 
-To display the Help for a function, type "get-help" followed by the function name. For example, to get Help for the Disable-PSRemoting function, type:
+Many scripts and functions in PowerShell have Help articles. Use the `Get-Help` cmdlet to
+display the Help articles for scripts and functions.
 
-```
-get-help disable-psremoting
+To display the Help for a function, type "Get-Help" followed by the function name. For example, to
```

Comment:
Should this Get-Help be code-fenced or in quotes?

Reply:
Done